types/mg-api-js/index.d.ts

```diff
@@ -0,0 +1,17735 @@
+             */
+            licenseState: string;
+            /**
+             * The device major firmware version. Newer versions have more functionality. Live device firmware versions are managed automatically. Default [0].
+             */
+            major: number;
+            /**
+             * The minimum accident speed in km/h. Default [4].
+             */
+            minAccidentSpeed: number;
+            /**
+             * The minimum allowable value for MaxSecondsBetweenLogs.
+             * For GoDevices, this is the same as the default value of 200.0f.
+             */
+            minSecondsBetweenLogs: object;
+            /**
+             * The device minor firmware version. Newer versions have more functionality. Live device firmware versions are managed automatically. Default [0].
+             */
+            minor: number;
+            /**
+             * The parameter version that is stored in MyGeotab.
+             * The parameter version should be increased by one when the parameters have been modified and need
+             * to be synced with the physical device. Default [2].
+             */
+            parameterVersion: number;
+            /**
+             * The parameter version that is on the device. Can be used to track the parameter version currently on the device by comparing to ParameterVersion. Default [0].
+             */
+            parameterVersionOnDevice: number;
+            /**
+             * The speeding off value in km/h. When 'IsSpeedIndicator' is enabled, once beeping starts, the vehicle must slow down to this speed for the beeping to stop. Default [90].
+             */
+            speedingOff: number;
+            /**
+             * The speeding on value in km/h. When 'IsSpeedIndicator' is enabled, the device will start beeping when the vehicle exceeds this speed. Default [100].
+             */
+            speedingOn: number;
+            /**
+             * The Vehicle Identification Number (VIN) of the vehicle associated with the device. Maximum length [50] Default [""].
+             */
+            vehicleIdentificationNumber: string;
+            /**
+             * Free text field where any user information can be stored and referenced for this entity.
+             */
+            comment: string;
+            /**
+             * The device features which have been enabled whether the feature is in use (e.g. HOS, Iridium).The DevicePlan property cannot be added/updated via the API.
+             */
+            deviceFlags: object;
+            /**
+             * Specifies the GO or Custom DeviceType.
+             */
+            deviceType: string;
+            /**
+             * The list of Group(s) the device belongs to.
+             */
+            groups: unknown[];
+            /**
+             * The unique hardware identifier for the Device. This is a 4 byte unique number to identify devic
```

Comment:
```suggestion
export as namespace GeotabApi;
```
https://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-d-ts.html#optional-global-usage
(using built-in, documented syntax, outcome is same)
were this generated?

Reply:
wow, GitHub lost track of the code snip :)
Skip this one, just was that code generated question stands